File: beginner/pagina01/1018/1018.cpp
#include <iostream>

using namespace std;

int main(){

	int value;

	cin >> value;

	cout << value << endl;

	cout << (value / 100) << " nota(s) de R$ 100,00" << endl;
	value %= 100;

	cout << (value / 50) << " nota(s) de R$ 50,00" << endl;
	value %= 50;

	cout << (value / 20) << " nota(s) de R$ 20,00" << endl;
	value %= 20;

	cout << (value / 10) << " nota(s) de R$ 10,00" << endl;
	value %= 10;

	cout << (value / 5) << " nota(s) de R$ 5,00" << endl;
	value %= 5;

	cout << (value / 2) << " nota(s) de R$ 2,00" << endl;
	value %= 2;

	cout << (value / 1) << " nota(s) de R$ 1,00" << endl;
	value %= 1;

	return 0;
}

File: beginner/pagina01/1018/1018.php
<?php
    $n = intval(fgets(STDIN));

    $cen = floor($n / 100);
    $rest100 = $n % 100;

    $cin = floor($rest100 / 50);
    $rest50 = $rest100 % 50;

    $vin = floor($rest50 / 20);
    $rest20 = $rest50 % 20;

    $dez = floor($rest20 / 10);
    $rest10 = $rest20 % 10;

    $cinco = floor($rest10 / 5);
    $rest5 = $rest10 % 5;

    $dois = floor($rest5 / 2);
    $rest2 = $rest5 % 2;

    $um = floor($rest2);

    echo "$n\n";
    echo "$cen nota(s) de R$ 100,00\n";
    echo "$cin nota(s) de R$ 50,00\n";
    echo "$vin nota(s) de R$ 20,00\n";
    echo "$dez nota(s) de R$ 10,00\n";
    echo "$cinco nota(s) de R$ 5,00\n";
    echo "$dois nota(s) de R$ 2,00\n";
    echo "$um nota(s) de R$ 1,00\n";
?>
